### PR TITLE
Create a Animal Table with postgresql

### PR DESCRIPTION
**Pull request by zunairkhan811**
- Database created

createdatabase

- Created Animals table

tablecreated

- Inserted data in Table

insertdata

- Different queries code has been written the screenshots of which are below

between2016and2019query

dateofbirthagumonorpikachuquery

monquery

neuteredandescapelessthan3query

neuteredquery

notgabumonquery

weightbetween10 4and17 3query

weightgreaterthan10 5query

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
+INSERT INTO animals VALUES (2, 'Gabumon', '2018-11-03', 2, true, 8);
+INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
+INSERT INTO animals VALUES (4, 'Devimon', '2017-05-23', 5, true, 11);
```
Comment (Davidosky007) [inline data.sql]: * Kindly correct the date of birth for these animals ( Gabumon, Devimon) as they are not correct 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @zunairkhan811 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Well written PR 👍 
* Tables created properly 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (zunairkhan811) [reply data.sql]: -Thanks for the suggestions. Your suggestion has been implemented.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @zunairkhan811,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**